[Bug] Flink doesn't support converting type BINARY(1) to Doris type yet

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.


### Version

<dependency>
      <groupId>org.apache.doris</groupId>
       <artifactId>flink-doris-connector-1.18</artifactId>
        <version>1.6.0</version>
</dependency>

### What's Wrong?

When I use flink cdc doris pipeline, a error happend, the stacktrace is like:  
Caused by: java.lang.RuntimeException: Failed to schema change, CreateTableEvent{tableId=DB.TABLE, schema=columns={`id` BIGINT NOT NULL,ARCHAR(50),`mock` BINARY(1)}, primaryKeys=id, options=()}, reason: Flink doesn't support converting type BINARY(1) to Doris type yet.
	at org.apache.flink.cdc.connectors.doris.sink.DorisMetadataApplier.applySchemaChange(DorisMetadataApplier.java:87)
	at org.apache.flink.cdc.runtime.operators.schema.coordinator.SchemaRegistryRequestHandler.applySchemaChange(SchemaRegistryRequestHandler.java:108)
	at org.apache.flink.cdc.runtime.operators.schema.coordinator.SchemaRegistryRequestHandler.lambda$flushSuccess$0(SchemaRegistryRequestHandler.java:196)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
 

### What You Expected?

Support converting type BINARY(1) of Flink to Doris type.

### How to Reproduce?

_No response_

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/flink-doris-connector/src/main/java/org/apache/doris/flink/catalog/DorisTypeMapper.java
+++ b/flink-doris-connector/src/main/java/org/apache/doris/flink/catalog/DorisTypeMapper.java
@@ -21,6 +21,7 @@ import org.apache.flink.table.api.DataTypes;
 import org.apache.flink.table.types.DataType;
 import org.apache.flink.table.types.logical.ArrayType;
 import org.apache.flink.table.types.logical.BigIntType;
+import org.apache.flink.table.types.logical.BinaryType;
 import org.apache.flink.table.types.logical.BooleanType;
 import org.apache.flink.table.types.logical.CharType;
 import org.apache.flink.table.types.logical.DateType;
@@ -237,6 +238,11 @@ public class DorisTypeMapper {
 
         @Override
         public String visit(RowType rowType) {
+            return STRING;
+        }
+
+        @Override
+        public String visit(BinaryType binaryType) {
             return STRING;
         }
 

--- a/flink-doris-connector/src/test/java/org/apache/doris/flink/catalog/DorisTypeMapperTest.java
+++ b/flink-doris-connector/src/test/java/org/apache/doris/flink/catalog/DorisTypeMapperTest.java
@@ -176,4 +176,10 @@ public class DorisTypeMapperTest {
                         DataTypes.ROW(DataTypes.FIELD("field", DataTypes.INT())));
         assertEquals("STRING", dorisType);
     }
+
+    @Test
+    public void testBinaryType() {
+        String dorisType = DorisTypeMapper.toDorisType(DataTypes.BINARY(1));
+        assertEquals("STRING", dorisType);
+    }
 }

